### PR TITLE
GitHub actions

### PR DESCRIPTION
**Pull request by kingsleyzissou**
- remove circle-ci 
- add publish command
- add GitHub actions config

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+      - 'performance/**'
+      - 'packages/**/*md'
+      - '.github/ISSUE_TEMPLATE/**'
+      - '.github/workflows/build-website.yml'
+
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
+    branches:
+      - master
+    paths-ignore:
```
Comment (wtrocki) [inline .github/workflows/ci.yml:24]: Not sure if we need to be such explicit on those

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+      - '.github/*.yml'
+      - 'performance/**'
+      - 'packages/**/*md'
+      - '.github/ISSUE_TEMPLATE/**'
+      - '.github/workflows/build-website.yml'
+
+  release:
+      types: [released, prereleased]
+
+  env:
```
Comment (wtrocki) [inline .github/workflows/ci.yml:40]: This should be removed

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+      if: steps.cache-dependencies.outputs.cache-hit != 'true'
+      run: yarn
+    - name: Build
+      run: yarn build
+    - name: Lint
+      run: yarn lint
+    - name: Unit Test
+      run: yarn test
+
+  publishPackages:
```
Comment (wtrocki) [inline .github/workflows/ci.yml:72]: I guess this is getting his realy bad flow from graphback. 
There are usually two files one for build one for release.
Doing both in one file seems overly complex

Comment (wtrocki) [reply .github/workflows/ci.yml:72]: https://github.com/redhat-developer/app-services-operator/tree/main/.github/workflows 

example

Comment (kingsleyzissou) [reply .github/workflows/ci.yml:72]: Wow that was quick. Haha this was still a work in progress

Comment (kingsleyzissou) [reply .github/workflows/ci.yml:72]: Cool, I'll have a look at that and maybe split it into two separate workflows

**Pull request merged**